packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -30,96 +30,131 @@
         <div
           v-if="showActions || selectedFiles.length > 0 || hasBulkActions"
           class="oc-flex-1 oc-flex oc-flex-start"
+          style="gap: 15px"
         >
           <template v-if="showActions && areDefaultActionsVisible">
+            <template v-if="createFileActionsAvailable">
+              <oc-button
+                id="new-file-menu-btn"
+                key="new-file-menu-btn-enabled"
+                v-oc-tooltip="newButtonTooltip"
+                :aria-label="newButtonAriaLabel"
+                variation="inverse"
+                :disabled="uploadOrFileCreationBlocked"
+                class="oc-background-primary-gradient"
+              >
+                <oc-icon name="upload" fill-type="line" />
+                <translate>Upload</translate>
```

Comment:
That's a rebase-gone-wrong here 🙄 not sure what exactly went wrong, but on it. Should be the "new" button